### DISCUSSION TITLE
Automatically add classes to blog post tags?

### DISCUSSION THREAD

**Discussion by OrigamiOrange** [wintercms/wn-blog-plugin#71] [Q&A] [answered]
In each blog post, the Blog plugin outputs plain `<p>` tags when `{{ post.content_html | raw }}` is called. Is it possible to add classes to those tags before they are printed to the page?

I know I can write, for example, `<p class="myclass">Blah blah blah</p>` in the edit post page within the CMS, but having to add a class (or multiple classes) manually for every `<p>` tag would get really tedious!

**Comment by mjauvin** [comment]
Other than using the `replace` twig filter, I don't think there is an easy way out of the box.

i.e.
```twig
{{ post.content_html | raw | replace({'<p>': '<p class="class1 class2">'}) }}
```

**Reply by OrigamiOrange** [reply, marked as answer]
> Other than using the `replace` twig filter, I don't think there is an easy way out of the box.
> 
> i.e.
> 
> ```
> {{ post.content_html | raw | replace({'<p>': '<p class="class1 class2">'}) }}
> ```

Thank you, this worked, albeit with a slight tweak. At first, it was outputting encoded HTML entities, so instead of `<p>` it would output `&lt;p&gt;`. But when I put the `raw` filter after `replace`, it worked! So the working solution looks like:
```
{{ post.content_html | replace({'<p>': '<p class="class1 class2">'}) | raw }}
```
which outputs the html.

Thanks so much!

**Comment by LukeTowers** [comment]
@OrigamiOrange you could also target the p tags in your CSS by wrapping the content in a class:

```twig
<div class="post-content">{{ post.content_html | raw }}</div>
```

```css
.post-content p {
    // styles here
}
```
